app/models/document_type.rb

```diff
@@ -21,6 +21,7 @@ def self.all
         hash["tags"] = hash["tags"].to_a.map(&TagField.method(:new))
         hash["publishing_metadata"] = PublishingMetadata.new(hash["publishing_metadata"].to_h)
         hash["guidance"] = hash["guidance"].to_a.map(&Guidance.method(:new))
+        hash["topics"] ||= true
```

Comment:
I'd have thought this ends up being set to true when ever it's false. 

Consider:

```
irb(main):004:0> hash = { topics: false }
=> {:topics=>false}
irb(main):005:0> hash[:topics] ||= true
=> true
irb(main):006:0> hash[:topics]
=> true
```
I may be missing something as I assume you'd have test failures?

Doing `hash[:topics] = hash.fetch(:topics, true)` would sort it though

Reply:
That confused me for a moment, too. It's because the tests all use the `document_type` factory, which initializes the model directly. But you're right the `||=` is redundant. I've updated to be more explicit - if it's still confusing I could add a comment to explain.

Reply:
Sure, yeah I think a comment would be useful as it's intentions will be somewhat cryptic.

Reply:
👍- updated.